sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -70,111 +60,46 @@ export interface OperationState<TResult> {
 export type OperationStatus = "notStarted" | "running" | "succeeded" | "canceled" | "failed";
 
 // @public
-export abstract class Poller<TState extends PollOperationState<TResult>, TResult> implements PollerLike<TState, TResult> {
-    constructor(operation: PollOperation<TState, TResult>);
-    cancelOperation(options?: {
-        abortSignal?: AbortSignalLike;
-    }): Promise<void>;
-    protected abstract delay(): Promise<void>;
-    getOperationState(): TState;
-    getResult(): TResult | undefined;
-    isDone(): boolean;
-    isStopped(): boolean;
+export interface PollerLike<TState extends OperationState<TResult>, TResult> extends Promise<TResult> {
+    readonly isDone: boolean;
+    readonly isStopped: boolean;
```

Comment:
Do we need this? I thought we're going to go with abort signal-based design for stopping.

Reply:
I think it would be useful to show if the aborted is success or not.

Reply:
As offline discussed this could be indicated if the result is undefined so we could remove this.